src/ConnectedMode/Migration/XmlFileCleaner.cs

```diff
@@ -53,7 +53,11 @@ public XmlFileCleaner(ILogger logger) : this(logger, new XmlDocumentHelper())
 
         public string Clean(string content, LegacySettings legacySettings, CancellationToken token)
         {
-            var document = xmlDocumentHelper.LoadFromString(content);
+            if (!xmlDocumentHelper.TryLoadFromString(content, out var document))
+            {
+                logger.WriteLine("Could not load xml");
```

Comment:
Normal verbosity string => should be in a resource file and should have a `[Migration]` prefix.
`The file could not be loaded as XML` might be a clearer message.

Could you fix the other string in the file at the same time please?